core/identity/src/service/identity.rs

```diff
@@ -55,11 +55,14 @@ fn send_event(s: Ref<Subscription>, event: model::event::Event) -> impl Future<O
 
     async move {
         for endpoint in subscriptions {
-            match bus::service(&endpoint).call(event.clone()).await {
-                Err(e) => log::error!("failed to send event: {}", e),
-                Ok(Err(e)) => log::error!("failed to send event: {}", e),
-                Ok(Ok(_)) => log::debug!("sent event: {:?} to {}", event, endpoint),
-            }
+            let msg = event.clone();
+            Arbiter::spawn(async move {
+                match bus::service(&endpoint).call(msg.clone()).await {
```

Comment:
Not sure, but cloning message here might be redundant. It's already been cloned 2 lines above.

Reply:
Can change logging order (not use moved value).